lib/pure/random.nim

```diff
@@ -233,28 +232,6 @@ proc rand*(r: var Rand; max: Natural): int {.benign.} =
     if x <= randMax - (randMax mod Ui(max)):
       return int(x mod (uint64(max)+1u64))
 
-proc rand*(max: int): int {.benign.} =
-  ## Returns a random integer in the range `0..max`.
-  ##
-  ## If `randomize<#randomize>`_ has not been called, the sequence of random
-  ## numbers returned from this proc will always be the same.
-  ##
-  ## This proc uses the default random number generator. Thus, it is **not**
-  ## thread-safe.
```

Comment:
What about this remark? Shouldn't we keep it?

Reply:
It is already kept
https://github.com/nim-lang/Nim/blob/de29bed82e40597a5d6450e697f06a9f9f87d16a/lib/pure/random.nim#L264